fah_alchemy/tests/unit/test_base_api.py

```diff
@@ -15,6 +26,8 @@ def tokendata():
             "org1-campaignA-projectI",
             "org1-campaignB-projectI",
             "org1-campaignB-projectII",
+            "org2-*-*",
+            "org3-campaignA-*",
```

Comment:
I added two additional `Scope`s to this user's authorized list. Can you:
1. fix the tests below that these additions broke, and
2. modify `base.api.validate_scopes` and `base.api.validate_scopes_query` such that they handle the presence of non-specific scopes such as these among a user's authorized set?
    - for `validate_scopes`, though `scope` may be specific, the `Scope`s in `token` won't necessarily be.
    - for `validate_scopes_query`, neither the `query_scope` or the `Scope`s in `token` will necessarily be specific.
    - for both these cases, it may make sense to add a `superset` method or similar to `Scope` that you can use for evaluating `Scope`s against each other. I've added a stub for this.

Reply:
@dotsdl I think this has the correct behaviour now. Feel free to take a look. :)